src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use ApiPlatform\Core\Api\IriConverterInterface;
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Client\ResponseCheckerInterface;
+use Sylius\Behat\Service\SharedStorageInterface;
+use Sylius\Component\Core\Model\CustomerInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Order\OrderTransitions;
+use Webmozart\Assert\Assert;
+
+final class ManagingOrdersContext implements Context
+{
+    /** @var ApiClientInterface */
+    private $client;
+
+    /** @var ResponseCheckerInterface */
+    private $responseChecker;
+
+    /** @var IriConverterInterface */
+    private $iriConverter;
+
+    /** @var SharedStorageInterface */
+    private $sharedStorage;
+
+    public function __construct(
+        ApiClientInterface $client,
+        ResponseCheckerInterface $responseChecker,
+        IriConverterInterface $iriConverter,
+        SharedStorageInterface $sharedStorage
+    ) {
+        $this->client = $client;
+        $this->responseChecker = $responseChecker;
+        $this->iriConverter = $iriConverter;
+        $this->sharedStorage = $sharedStorage;
+    }
+
+    /**
+     * @When I browse orders
+     */
+    public function iBrowseOrders(): void
+    {
+        $this->client->index();
+    }
+
+    /**
+     * @When I view the summary of the order :order
+     */
+    public function iSeeTheOrder(OrderInterface $order): void
+    {
+        $this->client->show($order->getNumber());
+    }
+
+    /**
+     * @When I cancel this order
+     */
+    public function iCancelThisOrder(): void
+    {
+        $this->client->applyTransition(
+            $this->responseChecker->getValue($this->client->getLastResponse(), 'number'),
```

Comment:
What will happen, if the previous message didn't contain order? I would recommend using `this order` to create a new uri.